frontend/packages/console-shared/src/components/markdown-extensions/showdown-extensions.ts

```diff
@@ -0,0 +1,168 @@
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import useCloudShellAvailable from '@console/app/src/components/cloud-shell/useCloudShellAvailable';
+import { MARKDOWN_COPY_BUTTON_ID, MARKDOWN_EXECUTE_BUTTON_ID, MARKDOWN_SNIPPET_ID } from './const';
+
+import './showdown-extension.scss';
+
+export const useInlineCopyClipboardShowdownExtension = () => {
```

Comment:
I'd prefer to have a single file per extension
`inline-clipboard-extension.ts`, `multiline-clipboard-extension.ts` etc...